### ISSUE TITLE
Better badge management

### ISSUE THREAD

**Issue by S-S-X** [mt-mods/mineunit-actions#3]
Without using repository for data storage. BYOB creates "shields" branch and pushes config there.

Also integrate badge management into action steps, related #1

**Comment by BuckarooBanzay** [comment]
(not sure if we talked about this, so i'm posting it here)
I found this recently: https://github.com/moteus/luacov-coveralls

**Comment by S-S-X** [comment]
> (not sure if we talked about this, so i'm posting it here)
> I found this recently: https://github.com/moteus/luacov-coveralls

Yes I looked at this and found out that there's been some problems with it.
Might work well, I just decided to no try it after looking around their site and some github projects using coveralls.

Could possibly actually try it out somewhere. My first impression just was that it would have a lot of cool features (no github integrations though) but also a lot of problems.
This first impression was solely based on quick browsing through github / google / their site, not actually trying out anything.
